sympy/solvers/tests/test_diophantine.py

```diff
@@ -124,7 +124,7 @@ def test_quadratic_elliptical_case():
     assert diop_solve(x**2 + y**2 + 2*x + 2*y + 2) == set([(-1, -1)])
     #assert diop_solve(15*x**2 - 9*x*y + 14*y**2 - 23*x - 14*y - 4950) == set([(-15, 6)])
     assert diop_solve(10*x**2 + 12*x*y + 12*y**2 - 34) == \
-        set([(-1, 2), (1, 1)])
+        set([(-1, -1), (-1, 2), (1, -2), (1, 1)])
```

Comment:
It is not necessary to denote all sets with braces, though it might have been convenient if their number was reasonable. It will suffice to make the change in those lines that have to be edited. I think it is ok to leave the other sets as is for now.